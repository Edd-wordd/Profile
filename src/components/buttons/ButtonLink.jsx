import React from 'react'
import { Grid, Link } from '@mui/material'
import { CustomButton, BtnWrapper } from '../styles/buttons/ButtonLink.styles'

function ButtonLink(props) {
  return (
    <div>
      <Grid component={BtnWrapper}>
        <Link href={props.linkOne}>
          <CustomButton size="large">{props.btnOneText}</CustomButton>
        </Link>
      </Grid>
    </div>
  )
}

export default ButtonLink
