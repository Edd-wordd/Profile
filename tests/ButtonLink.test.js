import React from 'react'
import testComponent from './testComponent'
import ButtonLink from '../src/components/buttons/ButtonLink'

testComponent(<ButtonLink />, 'ButtonLink')
