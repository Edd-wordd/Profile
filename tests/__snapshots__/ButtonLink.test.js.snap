// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonLink should render ButtonLink 1`] = `"Component"`;
